packages/calypso-e2e/src/jest-playwright-config/environment.ts

```diff
@@ -438,6 +438,15 @@ function setupBrowserProxyTrap( browser: Browser ): Browser {
 						} )
 					);
 
+					// Remove the Reader popover if it exists, to prevent the element we want to click
+					// from being covered by it so that the element isn't clickable.
+					await page.evaluate( () => {
+						const readerPopover = document.querySelector( '.masterbar__reader-popover' );
+						if ( readerPopover ) {
+							readerPopover.remove();
+						}
+					} );
+
```

Comment:
I wouldn't recommend doing this. Ideally, e2e tests should reflect what the users see and what they experience. If we allow e2e tests to diverge that much, they could become disconnected from reality and eventually be useless.

Let's think from the perspective of the user instead, and edit the e2e test to reflect that.

What would the user do? Would they close the popover if it's visible, and then do what they wanted to do initially?

Reply:
Thanks for raising this! I'm trying out limiting the scope of pages the popover shows up on, since I imagine if you were trashing a post or accessing stats you wouldn't want to have to dismiss an unrelated tooltip.